x/auth/tx/config.go

```diff
@@ -54,22 +55,15 @@ var DefaultSignModes = []signingtypes.SignMode{
 	signingtypes.SignMode_SIGN_MODE_DIRECT,
 	signingtypes.SignMode_SIGN_MODE_DIRECT_AUX,
 	signingtypes.SignMode_SIGN_MODE_LEGACY_AMINO_JSON,
```

Comment:
You explained it in the issue description, but I would add a comment here to explain why it is not part of the `DefaultSignModes`.

Reply:
Will do